tools/ci_build/github/linux/docker/scripts/install_ubuntu.sh

```diff
@@ -42,6 +42,7 @@ PACKAGE_LIST="autotools-dev \
 	libssl1.1 \
 	libkrb5-3 \
 	libtinfo-dev \
+	libtinfo5 \
```

Comment:
Any way to use libtinfo-dev instead of libtinfo5 for tvm?

Reply:
When running `TVM EP` CI tests for PR, we got the following error:
```Shell
[ 62%] Linking CXX shared library libtvm.so
/usr/bin/ld.lld: error while loading shared libraries: libtinfo.so.5: cannot open shared object file: No such file or directory
collect2: error: ld returned 127 exit status
make[2]: *** [_deps/tvm-build/CMakeFiles/tvm.dir/build.make:1395: _deps/tvm-build/libtvm.so] Error 1
make[1]: *** [CMakeFiles/Makefile2:3262: _deps/tvm-build/CMakeFiles/tvm.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
```
From this error it follows that when linking `libtvm.so`, we are missing `libtinfo.so.5`.

This looks strange, because locally this problem is not reproduced, it is present only in ORT CI.
It looks like installing package `libtinfo-dev` installs `libtinfo.so.6`, but we need `libtinfo.so.5`.

So we have to install `libtinfo.so.5` as a separate package.